cms/static/js/views/utils/move_xblock_utils.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Provides utilities for move xblock.
+ */
+define(['jquery', 'underscore', 'common/js/components/views/feedback_alert', 'js/views/utils/xblock_utils',
+        'js/views/utils/move_xblock_utils', 'edx-ui-toolkit/js/utils/string-utils'],
+    function($, _, AlertView, XBlockViewUtils, MoveXBlockUtils, StringUtils) {
+        'use strict';
+        var undoMoveXBlock, showMovedNotification, hideMovedNotification;
+
+        undoMoveXBlock = function(data) {
+            XBlockViewUtils.moveXBlock(data.sourceLocator, data.sourceParentLocator, data.targetIndex)
+            .done(function(response) {
+                // show XBlock element
+                $('.studio-xblock-wrapper[data-locator="' + response.move_source_locator + '"]').show();
+                showMovedNotification(
+                    StringUtils.interpolate(
+                        gettext('Move cancelled. "{sourceDisplayName}" has been moved back to its original ' +
+                            'location.'),
+                        {
+                            sourceDisplayName: data.sourceDisplayName
+                        }
+                    )
+                );
+            });
+        };
+
+        showMovedNotification = function(title, data) {
+            var movedAlertView;
+            if (data) {
```

Comment:
it's because for undo move cancelled notification, we are not proving any extra data.